scripts/sentinel-visor.service

```
@@ -1,15 +1,15 @@
 [Unit]
-Description=Sentinel Chainwatch
+Description=Sentinel Visor Indexer
 After=lotus-daemon.service
 Requires=lotus-daemon.service
 
 [Service]
-Environment=GOLOG_FILE="/var/log/sentinel/chainwatch.log"
+Environment=GOLOG_FILE="/var/log/sentinel/visor-indexer.log"
 Environment=GOLOG_LOG_FMT="json"
```

```suggestion
Environment=GOLOG_LOG_FMT="json"
Environment=GOLOG_OUTPUT="file+stderr"
```
This is so that the logs go to both the file and to `journald`